docs/changehistory/NextVersion.md

```diff
@@ -52,3 +52,21 @@ The [FloatingViewportContent]($appui-react) component has been added to support
 ## Font Workspaces
 
 It is now possible to store and load fonts from a Font Workspace. See [Fonts]($docs/learning/backend/Fonts.md) for more details.
+
```

Comment:
Add an announcement that the quantity formatter APIs have been promoted from beta. Describe what can be done with them. If samples, sandboxes, or example code exist, link to them.

Reply:
Added info and linked to learning doc.